src/normalize/entities.ts

```diff
@@ -191,6 +191,17 @@ const AttachmentsInfo = new Entity({
 	mp: ['existingAttachments', ExistingAttachmentsInfo]
 });
 
+
+const DocumentIds = new Entity({
+	id : 'id'
+})
+
+const AttachmentsAndDocumentsInfo = new Entity({
+	aid: 'attachmentId',
+	doc: ['documents', DocumentIds] ,
```

Comment:
I am not sure but Is it required to use normalizer for `DocumentIds`? Anyway its returning id for id.

Reply:
as we are returuning id for id i dont think so